tests/tool/dockerfilelint_tool_plugin/test_dockerfilelint_tool_plugin.py

```diff
@@ -116,9 +116,51 @@ def test_dockerfilelint_tool_plugin_parse_valid():
 def test_dockerfilelint_tool_plugin_parse_invalid():
     """Verify that invalid output of dockerfilelint is ignored."""
     plugin = setup_dockerfilelint_tool_plugin()
-    output = "invalid text"
-    issues = plugin.parse_output(output)
-    assert not issues
+    output = "invalid text\n"
+    issues = plugin.parse_output([output])
+    assert len(issues) == 1
+    assert issues[0].filename == "EXCEPTION"
+    assert issues[0].line_number == "0"
+    assert issues[0].tool == "dockerfilelint"
+    assert issues[0].issue_type == "ValueError"
+    assert issues[0].severity == "5"
+    assert issues[0].message == "Expecting value: line 1 column 1 (char 0), on line: invalid text"
+
+
+def test_dockerfilelint_tool_plugin_scan_invalid_rc_file():
+    """Integration test: Make sure the dockerfilelint output hasn't changed."""
+    plugin = setup_dockerfilelint_tool_plugin(package="invalidrc_package")
+    if not plugin.command_exists("dockerfilelint"):
+        pytest.skip("Missing dockerfilelint executable.")
+    package = Package(
+        "invalidrc_package", os.path.join(os.path.dirname(__file__), "invalidrc_package")
+    )
+    package["dockerfile_src"] = [
+        os.path.join(os.path.dirname(__file__), "invalidrc_package", "Dockerfile")
+    ]
+    issues = plugin.scan(package, "level")
+    # Expecting 1 issue for each of the following lines:
```

Comment:
I'm not sure if I should include the full stack trace expected here, but did so for completeness to start. Let me know if you think it would be better to remove it.

Reply:
I like it in there. It communicates the expectation and intent of the unit test. Makes it easier for someone else (or future you) to confidently modify the code in the future, which is one of the benefits of unit testing.